app/models/lead.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
+# Class that holds lead information
```

Comment:
Do you think this is an important comment?

Reply:
I was forced by rubocop